contracts/validator-registration-contract/.soliumignore

```diff
@@ -1 +0,0 @@
-node_modules
```

Comment:
Why did you delete this? Can you revert?

Reply:
+1, I don't think we have node_modules directory with bazel, but I still wonder what the motivation for this was.

Is it because we don't have node_modules with bazel?